src/js/views/DownloadButtonView.js

```diff
@@ -45,42 +45,50 @@ define(['jquery', 'underscore', 'backbone', 'models/SolrResult'],
 						.attr("data-placement", "top")
 						.attr("data-trigger", "hover")
 						.attr("data-container", "body");
+
+				// Removing pointer as cursor and setting to default
+				this.$el.css("cursor","default");
 			}		
 		},
 		
 		download: function(e){
 			e.preventDefault();
 						
-			if(this.$el.is(".in-progress"))
-				return true;
-			
-			//Show that the download has started
-			this.$el.addClass("in-progress");
-			var buttonHTML = this.$el.html();
-			this.$el.html("Downloading... <i class='icon icon-on-right icon-spinner icon-spin'></i>");
-			
-			//If we found a model, fire the download event
-			this.model.downloadWithCredentials();
-			
-			this.listenTo(this.model, "downloadComplete", function(){
+			// Checking if the Download All button is disabled because the package is too large
+			var isDownloadDisabled = (this.$el.attr("disabled") == "disabled") ? true : false;
```

Comment:
It'd be good to use type stable comparison here, `===` (and everywhere, generally-speaking). What do you think about changing this from `==` to `===`?